Integrations/integration-MISP.yml

```diff
@@ -417,6 +423,92 @@ script:
     }
 
 
+    var addMinutes = function(date, minutes) {
+                return new Date(date.getTime() + minutes*60000);
+    };
+
+
+    var getNowTime = function() {
+        var time = new Date();
+        time = addMinutes(time,time.getTimezoneOffset());
+        var month = "0" + (time.getMonth()+1);
+        var day = "0" + time.getDate();
+        var hours = "0" + time.getHours();
+        var minutes = "0" + time.getMinutes();
+        var seconds = "0" + time.getSeconds();
+        var dateOld = time.getFullYear()+'-'+month.substr(-2)+'-'+day.substr(-2);
+        return dateOld;
+    };
+
+
+    function createEvent(command) {
+        var currentArgs = getProvidedArgsForCommand(command, args);
+        var ec = {};
+
+        if (currentArgs.to_ids === 'true') {
+            currentArgs.to_ids = true;
+        } else {
+            currentArgs.to_ids = false;
+        }
+
+        if (args.published === 'true') {
+            args.published = true;
+        } else {
+            args.published = false;
+        }
+
+        var requestUrl = params.url + commands[command].url;
+
+        var body = { 'Event': {
+            'date': getNowTime(),
+            'threat_level_id': '1',
+            'published': args.published,
+            'info': args.info,
+            'analysis': '0',
+            'distribution': '0',
+            'Attribute': [currentArgs] } };
+
```

Comment:
are you sure about all of those constants?  
'threat_level_id': '1'
 'analysis': '0'
 'distribution': '0'

maybe some of them needs to be arguments to the command?

Reply:
Yes, I checked it at the customer

Reply:
Fixed